# for publishing errors
# 1. cross track error
# 2. yaw error
# 3. speed error

std_msgs/Header header

float64 cross_track_error

float64 yaw_error_deg

float64 target_speed_ms
float64 current_speed_ms

